packages/core/src/sync-store/sqlite/store.ts

```diff
@@ -899,6 +903,244 @@ export class SqliteSyncStore implements SyncStore {
     });
   };
 
+  insertFactoryTraceFilterInterval = async ({
+    chainId,
+    factory,
+    block: rpcBlock,
+    transactions: rpcTransactions,
+    transactionReceipts: rpcTransactionReceipts,
+    traces: rpcTraces,
+    interval,
+  }: {
+    chainId: number;
+    factory: FactoryCallTraceFilterCriteria;
+    block: RpcBlock;
+    transactions: RpcTransaction[];
+    transactionReceipts: RpcTransactionReceipt[];
+    traces: SyncCallTrace[];
```

Comment:
Any reason this isn't called `RpcCallTrace`?

Reply:
The `Rpc` prefixed types are all from viem